package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@icure/medical-device-sdk",
-  "version": "2.0.6",
+  "version": "2.1.0",
```

Comment:
**Do not change the version manually: Use GitFlow**
To create a release, you'll first have to merge this PR in the develop branch and then create a release/branch. On this release/branch (that will also need a review), you'll have to include git tags to make sure your release will be created as expected. You can check the documentation here: https://internal.icure.dev/003_procedures/repo_release_management/#how-do-i-trigger-a-release

Reply:
resolved